scripts/api/common.sh

```diff
@@ -6,9 +6,15 @@ plain_rh_identity="{\"identity\":{\"account_number\":\"${ACCOUNT_NUMBER}\",\"use
 
 # base64 can split output to more lines, forbid it with one of following
 # depends on base64 version:
-export X_RH_IDENTITY=$(echo ${plain_rh_identity} | base64 --wrap=0)
-# export X_RH_IDENTITY=$(echo ${plain_rh_identity} | base64)
-# export X_RH_IDENTITY=$(echo ${plain_rh_identity} | base64 --break=0)
+if base64 --wrap=0 < /dev/null 2> /dev/null > /dev/null
+then
```

Comment:
Line 9 - 17 is this a OSX issue you ran into?  Might need more documentation around what this is fixing.

Reply:
yeah, I think OSX doesn't have `--wrap` argument. @lpichler which format are you using there?

Reply:
I think the original comment describes the issue fairly well:
```
# base64 can split output to more lines, forbid it with one of following
# depends on base64 version:
```
In the old version, the user had to know to uncomment the proper line of code, I just automated that. While technically it's more of a `base64` version issue, I think most users encountering this problem will be on OSX.